src/index.ts

```diff
@@ -464,6 +496,39 @@ if (process.env.NODE_ENV === 'production') {
   return fs.writeFile(resolveApp(`./dist/index.js`), contents);
 }
 
+function getAuthorName() {
+  let author = '';
+
+  author = shell
+    .exec('npm config get init-author-name', { silent: true })
+    .stdout.trim();
+  if (author) return author;
+
+  author = shell
+    .exec('git config --global user.name', { silent: true })
+    .stdout.trim();
+  if (author) {
+    setAuthorName(author);
```

Comment:
if they did not have an author with npm but do have a user name in git, we set their npm author name to it.